src/app/client/src/app/modules/core/components/main-footer/main-footer.component.spec.ts

```diff
@@ -90,4 +113,19 @@ describe('MainFooterComponent', () => {
         expect(spy).toHaveBeenCalledWith('https://play.google.com/store/apps/details?id=in.gov.diksha.app&referrer=utm_source=' +
         TestBed.get(ResourceService).instance + '-sunbird&utm_medium=');
     });
+
+    it('should redirect to playstore url provided from tenant config', () => {
+        const spy = spyOn(component, 'redirect');
+        component.tenantFooter.playstoreLink = 'https://play.google.com/store/apps/details?id=in.gov.diksha.app';
+        component.redirectToDikshaApp();
+        expect(spy).toHaveBeenCalledWith('https://play.google.com/store/apps/details?id=in.gov.diksha.app');
+    });
+
+    it('should call tenant service to get tenant config', () => {
+        spyOnProperty(userService, 'slug', 'get').and.returnValue('sunbird');
+        spyOn(component.tenantService, 'getTenantConfig').and.returnValue(of(response));
```

Comment:
Where is the mock data of the response object structure? That is the key thing here.

Reply:
- refer to https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/3878/files#diff-9786546627137d7552667d88ea06a2aeR45